### PR TITLE
fix(ComponentDemo): add padding to align dropdown labels

### PR DESCRIPTION
**Pull request by tw15egan**
Temporary workaround for https://github.com/carbon-design-system/carbon-website/issues/1688 until we implement the fluid versions of all form components. 

#### Changelog

**Changed**

- `1rem` margin to labels on `ComponentDemo` dropdowns to align the text
- Updated copy per @janchild / @aagonzales

### CODE REVIEW

File: src/components/ComponentDemo/ComponentDemo.js [hunk 1]
```diff
@@ -111,10 +111,8 @@ const ComponentDemo = ({ children, src, scope, noInline, components }) => {
             light
             initialSelectedItem={components[0]}
             id="component-variant"
-            titleText={
-              childrenArray.length === 1 ? '' : 'Component variant selection'
-            }
-            label="Component variant selection"
+            titleText={childrenArray.length === 1 ? '' : 'Variant selector'}
+            label="Variant selection"
```
Comment (janchild) [inline src/components/ComponentDemo/ComponentDemo.js:115]:
```suggestion
            label="Variant selector"
```

**Review by janchild — COMMENTED** [review]
@tw15egan One tweak! :)

File: src/components/ComponentDemo/ComponentDemo.js (hunk 1 above)
Comment (tw15egan) [reply src/components/ComponentDemo/ComponentDemo.js:115]: 🤦

**Review by aagonzales — APPROVED** [review]
Perfect! Thanks for updating!

**Review by emyarod — APPROVED** [review]

**Review by aledavila — APPROVED** [review]

**Pull request merged**